pkgs/by-name/la/lastfm/package.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, fetchzip
+, stdenvNoCC
+, writeShellApplication
+, curl
+, xmlstarlet
+, common-updater-scripts
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "lastfm";
+  version = "2.1.39";
+
+  src = fetchzip {
+    extension = "zip";
+    name = "Last.fm.app";
+    url = "https://cdn.last.fm/client/Mac/Last.fm-${finalAttrs.version}.zip";
+    hash = "sha256-jxFh0HbY4g5xcvAI20b92dL1FRvRqPwBBa0Cv9k63+s=";
+  };
+
+  dontConfigure = true;
+  dontBuild = true;
+
+  sourceRoot = ".";
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p "$out/Applications"
+    cp -r *.app "$out/Applications"
+
+    runHook postInstall
+  '';
+
+  passthru.updateScript = lib.getExe (writeShellApplication {
+    name = "lastfm-update-script";
+    runtimeInputs = [
+      curl
+      xmlstarlet
+      common-updater-scripts
+    ];
+    text = ''
+      url=$(curl --silent "https://cdn.last.fm/client/Mac/updates.xml")
+      version=$(echo "$url" | xmlstarlet sel -t -v "substring-before(substring-after(//enclosure/@url, 'version='), '&')")
+      update-source-version lastfm "$version" --file=./pkgs/by-name/la/lastfm/package.nix
```

Comment:
is `--file` really necessary?

Reply:
The `--file` part was from a template that someone else had me use on a previous pr that I wrote, I looked into the code of update-source-version and looks like its able to find the file from meta.position. Removed the --file part.